### DISCUSSION TITLE
Could not find the Qt platform plugin "windows" in ""

### DISCUSSION THREAD

**Discussion by hymancheng** [pyinstaller/pyinstaller#5940] [Help]
I used 'upx' in the pyinstaller command, when I execute exe file. 
The error is as follows:
qt.qpa.plugin: Could not find the Qt platform plugin "windows" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

When I remove 'upx' command, exe is running normally.

**Comment by rokm** [comment]
That's a known issue with `upx` - see upx/upx#107. So you'll either need to disable `upx`altogether, or add all (used) Qt plugins to `upx_exclude`, or use `develop` branch, which should automatically prevent Qt plugins from being processed with `upx` (see #5881).
